projects/plugins/jetpack/changelog/fix-google-photos-disconnect

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: other
+
+Fixes a bug on WP installations that are not fully updated to 6.2.0 yet.
```

Comment:
That's either sites that installed a five-month-old 6.2 alpha version and then haven't updated since for some reason, or WordPress.com Simple that's apparently slowly updating commit-by-commit and hasn't gotten to [54997](https://core.trac.wordpress.org/changeset/54997) yet.

Might be clearer to just say "WordPress.com Simple" since I doubt there are any sites people care about in the first category.